app/graphql/types/enum/installment_tag.rb

```diff
@@ -0,0 +1,4 @@
+class Types::Enum::InstallmentTag < Types::Enum::Base
+  value 'SIDE_STORY', '', value: :side_story
+  value 'SPINOFF', '', value: :spinoff
```

Comment:
May need more installment tags. Some suggested ones (just following the same syntax here)
`value 'MAIN_ENTRY', '', value: :main_entry`
`value 'CROSSOVER', '', value: :crossover`
`value 'ALTERNATE_SETTING', '', value: :alternate_setting`

Reply:
You can probably base it off of these. Not all of them are relevant but some might be

![](https://i.imgur.com/WBWWLes.jpg)